config/ompi_configure_options.m4

```diff
@@ -166,6 +166,16 @@ case "x$enable_mpi_fortran" in
         ;;
 esac
 
+AC_ARG_ENABLE([force-gcc-atomics],[AS_HELP_STRING([--enable-force-gcc-atomics],
```

Comment:
`--enable-force-gcc-atomics` is meant for Power only, so it would be nice to have that in the name. That way someone doesn't think it applies to their platform without reading the help message.

Maybe:
 * `--enable-gcc-atomics-for-ppc`
 * `--enable-builtin-pcc-atomics` (and flip the logic)

Reply:
Yeah this is a good idea. Done. Thanks